src/pages/wallets/find-wallet.tsx

```diff
@@ -203,86 +163,50 @@ const FindWalletPage = () => {
         </Center>
       </Flex>
 
-      <Hide above="lg">
-        <Box
-          display={{ base: "block", lg: "none" }}
-          position="sticky"
-          top={NAV_BAR_PX_HEIGHT}
-          bg="background.base"
-          w="full"
-          zIndex="docked"
-          py="5px"
+      {/* Wallet Personas */}
+      <Box px={{ base: 4, "2xl": 0 }}>
+        <OldHeading
+          as="h3"
+          fontSize="xl"
+          fontWeight="bold"
+          lineHeight={1.4}
+          mt={0}
+          mb={3}
         >
-          <Button
-            rightIcon={<FilterBurgerIcon />}
-            variant="outline"
-            borderInlineStart="none"
-            borderInlineStartRadius="none"
-            gap={4}
-            sx={{
-              svg: {
-                boxSize: 8,
-                line: { stroke: "primary.base" },
-                circle: { stroke: "primary.base" },
-              },
-            }}
-            onClick={() => {
-              showMobileSidebar ? onClose() : onOpen()
-              trackCustomEvent({
-                eventCategory: "MobileFilterToggle",
-                eventAction: `Tap MobileFilterToggle`,
-                eventName: `show mobile filters ${!showMobileSidebar}`,
-              })
-            }}
-          >
-            <Box>
-              <Text>{t("page-find-wallet-filters")}</Text>
-              <Text fontSize="sm" lineHeight="14px" color="body.medium">
-                {Object.values(filters).reduce(
-                  (acc, filter) => (filter ? acc + 1 : acc),
-                  0
-                )}{" "}
-                {t("page-find-wallet-active")}
-              </Text>
-            </Box>
-          </Button>
-        </Box>
-        <Drawer
-          isOpen={showMobileSidebar}
-          placement="start"
+          {t("page-find-wallet-personas-title")}
+        </OldHeading>
+
+        <WalletFilterPersona
+          resetFilters={resetFilters}
+          setFilters={setFilters}
+          selectedPersona={selectedPersona}
+          setSelectedPersona={setSelectedPersona}
+          showMobileSidebar={showMobileSidebar}
+        />
+      </Box>
+
+      {/* Mobile filters menu */}
+      <Hide above="lg">
```

Comment:
Similar to a previous comment, I'm getting a glitch where this displays on Desktop on load, then quickly disappears... recurring issue when using the `Show` or `Hide` components which are rendered server-side before it knows what size the device is. Switching to a Box (or other Chakra component) with `hideFrom="lg"` should fix this.